wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/DocumentInfoScreen.kt

```diff
@@ -246,24 +255,29 @@ fun DocumentInfoScreen(
                     text = { Text(text = stringResource(R.string.document_info_screen_menu_item_delete)) },
                     leadingIcon = { Icon(Icons.Outlined.Delete, contentDescription = null) },
                     onClick = {
-                        showMenu = false
+                        showMenu = false;
                         showDeleteConfirmationDialog = true
                     }
                 )
+                DropdownMenuItem(
+                    text = { Text(text = stringResource(R.string.document_info_screen_menu_activities_log)) },
+                    leadingIcon = { Icon(Icons.Outlined.Info, contentDescription = null) },
+                    onClick = {
+                        onNavigate(WalletDestination.DocumentInfo.getRouteWithArguments(
+                            listOf(
+                                Pair(WalletDestination.DocumentInfo.Argument.DOCUMENT_ID, documentInfo.documentId),
+                                Pair(WalletDestination.DocumentInfo.Argument.SECTION, "activities")
+                            )
+                        ))
+                        showMenu = false
+                    }
+                )
                 if (settingsModel.developerModeEnabled.value == true) {
                     Divider()
                     DropdownMenuItem(
                         text = { Text(text = stringResource(R.string.document_info_screen_menu_item_request_update)) },
-                        leadingIcon = {
-                            Icon(
-                                painter = painterResource(id = R.drawable.experiment_icon),
-                                contentDescription = null
-                            )
-                        },
-                        onClick = {
-                            showMenu = false
-                            showRequestUpdateDialog = true
-                        }
+                        leadingIcon = { Icon(painter = painterResource(id = R.drawable.experiment_icon), contentDescription = null) },
+                        onClick = { showMenu = false; showRequestUpdateDialog = true }
```

Comment:
Why are you reformatting existing code for no good reason? Same elsewhere.

Reply:
I am not intentionally going around and reformatting for no good reason. I am not sure how this line in specific got reformatted but it is possible that auto merge/ide undo/redo caused this.

Reply:
Reformatting undone.